src/Components/Blazor/Build/test/BuildIntegrationTests/Assert.cs

```diff
@@ -0,0 +1,950 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
```

Comment:
This is taken pretty much as-is from RazorSDK (along with some other types that were added). We need to share the code, but I don't want to embark on that just as yet with our current branching. Feel free to skip this type.